charts/redis/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: redis
+description: A Helm chart to deploy the redis for dev and test environment
+type: application
+# HelmChart Version
+version: 1.0.1
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 5.0.5
+home: https://github.com/whoacademy/helm-charts/charts/redis
+maintainers:
+  - name: springdo
+  - name: gkmahendran09
+  - name: rh-jpoole
```

Comment:
You can remove and add only your GitHub name here.

Reply:
It's updated